code/build.R

```diff
@@ -399,7 +399,7 @@ nrow(r_lanes_top)
 r_lanes_top %>% sf::st_drop_geometry()
 
 # classify roads to visualise
-labels = c("Top route", "Spare lane(s)", "Estimated width > 10m")
+labels = c("Spare lane(s)", "Estimated width > 10m", "Top route")
```

Comment:
Looks good, definite improvement. I wonder, can the 'Top route' go top in the legend *and* top in terms of map layers?